ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/UpdatePermanentIdRequest.ui.xml

```diff
@@ -19,9 +20,16 @@
 
     <container:VerticalLayoutContainer>
         <container:child layoutData="{layoutData}">
+            <form:FieldLabel text="Email">
```

Comment:
This is minor but you could probably move the "Email" text to the appearance, like you have in the next FieldLabel for consistency.